conans/test/conan_v2/test_default_config.py

```diff
@@ -0,0 +1,35 @@
+import platform
+import unittest
+
+from conans.test.utils.conan_v2_tests import ConanV2ModeTestCase
+
+
+class DefaultConfigTestCase(ConanV2ModeTestCase):
+    def test_revisions_enabled(self):
+        t = self.get_client()
+        self.assertEqual(t.cache.config.revisions_enabled, True)
+        t.run('config get general.revisions_enabled')
+        self.assertEqual(str(t.out).strip(), "1")
```

Comment:
Remember this is not enough. It should raise if the user had a previous config with revisions_enabled=0